Mixpanel/Mixpanel.swift

```diff
@@ -30,8 +30,8 @@ public class Mixpanel {
      You can always get the instance by calling getInstance(name)
      */
     @discardableResult
-    public class func initialize(token apiToken: String,
-                                                    launchOptions: [NSObject: AnyObject]? = nil,
+    open class func initialize(token apiToken: String,
```

Comment:
In the spirit of this being a public SDK, I wanted to leave everything open, but I'm happy to change it back to public if you think differently